lldb/source/Plugins/ObjectFile/PECOFF/ObjectFilePECOFF.cpp

```diff
@@ -355,11 +355,9 @@ size_t ObjectFilePECOFF::GetModuleSpecifications(
 }
 
 bool ObjectFilePECOFF::SaveCore(const lldb::ProcessSP &process_sp,
-                                const lldb_private::FileSpec &outfile,
-                                lldb::SaveCoreStyle &core_style,
+                                const lldb_private::CoreDumpOptions &core_options,
                                 lldb_private::Status &error) {
-  core_style = eSaveCoreFull;
```

Comment:
we are chanigng the default style of this SaveCore function, there might be tests that will fail on Windows if we don't fix the test. Ignore this if you changed the test already or if there wasn't one.

Reply:
For this function COFF never actually references the corestyle in saveminidump, so I am resolving.